### PR TITLE
Sentinel-2 Harmonization & fix L8 harm & fix sen2cor output

### PR DESCRIPTION
**Pull request by marujore**
New: Sentinel-2 Harmonization
Fix: Landsat-8 Harmonization file names
Fix: Sen2cor output dir

### CODE REVIEW

File: bdc_collection_builder/collections/landsat/harmonization.py [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
-from bdc_collection_builder.collections.utils import load_img
-from bdc_collection_builder.collections.models import RadcorActivity
+
+# BDC DB
 from bdc_db.models import CollectionItem
 
+# BDC Scripts
+from ...config import Config
+from ..nbar import process_NBAR
+from ..utils import load_img
```
Comment (raphaelrpl) [inline bdc_collection_builder/collections/landsat/harmonization.py:15]: Please, sort imports alphabetically

File: bdc_collection_builder/collections/sentinel/tasks.py [hunk 2]
```diff
@@ -277,6 +280,74 @@ def upload(self, scene):
...
+            dir_published_L2 = str(Path(Config.DATA_DIR) / 'Repository/Archive/{}/{}/{}/'.format('S2SR_SEN28', yyyymm, os.path.basename(SAFEL2A)))
+            L1_dir = str(Path(Config.DATA_DIR) / 'Repository/Archive/{}/{}/'.format('S2_MSI', yyyymm))
+            zip_pattern = '.*_MSIL1C_{}{}{}.*_{}_.*.zip$'.format(y,m,d,mgrs)
+            zip_file_name = [os.path.join(L1_dir,d) for d in os.listdir(L1_dir) if re.match('{}'.format(zip_pattern), d)][0]
+
+            # Check if file is valid
+            valid = is_valid(zip_file_name)
+
+            if not valid:
+                raise IOError('Invalid zip file "{}"'.format(zip_file_name))
```
Comment (raphaelrpl) [inline bdc_collection_builder/collections/sentinel/tasks.py:325]: This section looks like the `download` step where we must check if zip is valid before extract.
Could you create a common function to these code?

File: bdc_collection_builder/collections/landsat/harmonization.py (hunk 1 above)
Comment (marujore) [reply bdc_collection_builder/collections/landsat/harmonization.py:16]: done

File: bdc_collection_builder/collections/sentinel/tasks.py (hunk 2 above)
Comment (marujore) [reply bdc_collection_builder/collections/sentinel/tasks.py]: done (extract_and_get_internal_name)

**Pull request merged**